### DISCUSSION TITLE
Author list

### DISCUSSION THREAD

**Discussion by rinka88** [templaza/astroid-framework#341] [Ideas]
The authors page is implemented in the T4 template out of the box.
[https://t4-framework.demo.joomlart.com/index.php/joomla-pages/bonus-pages/authors](https://t4-framework.demo.joomlart.com/index.php/joomla-pages/bonus-pages/authors)

If there were such on Astroid, it would be great for news templates

**Comment by Chacapamac** [comment]
I @rinka88 
This is not really an issue, more a discussion. Issues are more for Astroid bugs. 
I transfer it to the “Discussions”.

I mark this as an “Enhancement”.

Good Day!

**Comment by rinka88** [comment]
Thank you. I just didn't know where to take it, so I wrote there. But I'm interested in whether such a template will be implemented. It would be great for news sites where there are many authors.

**Comment by rinka88** [comment]
By the way, here are his setup instructions. there they included the overrides in the template plugin.
https://www.joomlart.com/documentation/t4-framework/author-page-settings

**Comment by Chacapamac** [comment]
@rinka88 ,
 I look fast and if I understand well, this is an Astroid feature that let you display an author block base on the User Astroid “Author Profile” option.
![edit-astroid-profile](https://github.com/templaza/astroid-framework/assets/1907276/e3304ae1-fcfc-41de-bee6-fa6966fbc2e0)

**What I know so far:**
• If you fill this option the info will appear in all articles made by this user
• If you want to turn it off, the only way I find is by the Article Astroid “Blog Option”
![article-astroid-blog-option](https://github.com/templaza/astroid-framework/assets/1907276/ebad0a51-b4aa-414b-9975-165d077ed1fd)

I see no way to control that possibility globally or by Menu items Astroid special option – ”Blog Option” — You have to turn it off at the article level????
![menu-blog-option](https://github.com/templaza/astroid-framework/assets/1907276/b4616b9b-56bb-409b-b0dc-f3e60e8688f6)

So far I see no way to build an “Astroid Multi Profiles page” ??????

Few problems here.....
I test more and come back....

**Comment by rinka88** [comment]
here we mean not only the Author Profile, and the list of authors-contacts, and when you click on the link, you get the page of the author-contact, where his information and below materials published on his behalf.

**Reply by Chacapamac** [reply]
@rinka88 , I try to see what is possible with **Astroid Author Profile**
You probably know more on what should be needed to make that function interesting and useful for user

**Can you help define what this function should have to be useful?** 

**This is my rapid overview, random suggestion and remarks**
_**Note:** I’m maybe wrong on some points, @rinka88  help me understand._
— It will be interesting to be able to use onboard editor in the ➔ **_Author About Me_** field
— No ability to choose a position of the Author info block (Top or bottom of the article)
— No Global configuration to define what we want when creating new articles
— No ability to turn on or off the possibility by **Menu Items** or by **type of articles** _(like blog)_
— No ability to create or inject an Author or Selected or All Authors info block in an article or module
— No link to a content list from a particular in the Authors info block

**Reply by rinka88** [reply]
when choosing a menu, there is a tab where you can select a page with authors - contacts. On the page with the authors - contacts, when you click on the name, you get to his page, and there his published materials. Such a page would be great for a news site that publishes materials from different authors.

**Reply by Chacapamac** [reply]
@sonvnn , when you have time, can you look at this to ameliorate what we can do with the “**Astroid Author Profile**” — Also, maybe, we don’t know how to use it correctly?

**Comment by Chacapamac** [comment]
@sonvnn 
Just adding a bug to the discussion
Trying this Astroid feature (Author Profile) I remove some Social links and try to put new ones in my  ➔ **Users: Edit Profile ➤ Author Profile** and I get this error.
![Screen Shot 2023-06-11 at 8 53 44 AM](https://github.com/templaza/astroid-framework/assets/1907276/855f5479-e84e-4c98-99c5-7b175bea50dc)

![Screen Shot 2023-06-11 at 8 57 44 AM](https://github.com/templaza/astroid-framework/assets/1907276/a3950107-c8ce-499a-b2dc-fa0224cb3512)

**Reply by rinka88** [reply]
the error is still visible

**Comment by paurray** [comment]
Not really a function that I currently need personally. But I think it would be a great feature. (It is one of the reasons that I used Easy Blog/Stackideas for years as they had/have a multi author option.) My 2 cents. Very cool.

**Comment by rinka88** [comment]
@sonvnn Excuse me, can I ask? Will there be a change in this topic in the next version?

**Reply by sonvnn** [reply]
This ideas will be available in Astroid 3.

Thank you :)

**Comment by rinka88** [comment]
Even offers from core joomla 5.
https://github.com/joomla/joomla-cms/pull/41132

I think it's worth considering.

**Comment by rinka88** [comment]
@sonvnn I still can’t implement what is described in this thread.

**Reply by rinka88** [reply]
maybe look here for ideas:
https://www.dr-menzel-it.de/blog/override-challenge-autor-seite

**Comment by rinka88** [comment]
@sonvnn Now all that remains is to fix the error in this section and, if possible, add the override suggested above)

**Reply by sonvnn** [reply]
Yeah! I will try to play with it after 3.0.15. May be it will be release in 3.0.16. Thank you.

**Reply by rinka88** [reply]
and don’t forget about this improvement, if possible )

**Reply by rinka88** [reply]
@sonvnn Don’t forget about this feature :)

**Reply by sonvnn** [reply]
@rinka88 Actually, You can see this page here https://demo.astroidframe.work/contact
Click to Victoria Lee you can see her articles https://demo.astroidframe.work/contact/victoria-lee
I did it follow https://github.com/joomla/joomla-cms/pull/41132

**Reply by rinka88** [reply]
It is perfectly. But if you want to display materials with pictures and publication dates, with the name of the category, as I wanted in the original example from Joomlart

**Reply by rinka88** [reply]
https://magazine.joomla.org/all-issues/july-2025/how-to-create-an-author-s-page

**Comment by Chacapamac** [comment]
Perfect!

**Comment by rinka88** [comment]
https://www.dr-menzel-it.de/blog/autor-seite-fuer-joomla-4

maybe this method works and will help someone

**Reply by sonvnn** [reply]
Thank you! I will refer to it and incorporate it into the application in upcoming versions.

**Comment by paurray** [comment]
Coming back to this one because it keeps proving useful. If you are trying to build a proper authors page, the good news is you can do it using core Joomla, no extra extension needed, and Astroid will handle the styling nicely.

The short version:

Link each author’s user account to a Contact (Contact edit, Linked User), and enable the Content - Contact plugin. You can also enable User - Contact Creation if you want contacts created automatically with users.

Turn on author linking in the article options.

Create a “List Contacts in a Category” menu item. This becomes your authors or team page. In the menu item settings under List Layouts, set Image to Show so avatars appear. The image itself is set in the contact record under the Images and Links tab.

Each author page will then display their bio along with the articles they have written. That part is handled in core via PR joomla/joomla-cms#41132.

With Astroid, the list is rendered as a clean card grid out of the box, no template overrides needed. If you want it closer to the JoomlArt style with intro images and dates on article lists, that only requires a small com_contact override. Viviana Menzel covered this nicely in the Joomla Magazine (July 2025, “How to create an author's page”).

Looking into putting together a proper walkthrough for this, covering the native setup first and then the Astroid polish, since a few people have asked. Will share when it is up. Fingers crossed!

**Comment by rinka88** [comment]
Yes, that's great. And Astroid can be used too. And it would be better to have a page style like Joomlart (with images, headers, and other data) + author list pages.
